### PR TITLE
Add page for the draft posts

### LINKED ISSUE #15: Create the page for draft posts
- [x] Create the form for draft posts 

- [x] Add publish button

- [x] Add the ability to delete posts

### PR DESCRIPTION
**Pull request by pavlenk0**
Close #15

### CODE REVIEW

File: blog/views.py [hunk 1]
```diff
@@ -20,7 +20,6 @@ def post_new(request):
         if form.is_valid():
             post = form.save(commit=False)
             post.author = request.user
-            post.published_date = timezone.now()
```
Comment (webknjaz) [inline blog/views.py:23]: Why did you remove it?

File: blog/templates/blog/post_draft_list.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% extends 'blog/base.html' %}
+
+{% block content %}
+    {% for post in posts %}
+        <div class="post">
+            <p class="date">created: {{ post.created_date|date:'d-m-Y' }}</p>
```
Comment (webknjaz) [inline blog/templates/blog/post_draft_list.html:6]: It is nice to wrap pipes with whitespaces

File: blog/views.py (hunk 1 above)
Comment (pavlenk0) [reply blog/views.py:23]: @webknjaz to save the post as a draft. New posts will be saved as drafts rather than being instantly published. When the draft post has been created, there is no need to know the exact time until this post will be publish.

File: blog/templates/blog/post_draft_list.html (hunk 2 above)
Comment (pavlenk0) [reply blog/templates/blog/post_draft_list.html:6]: Ok. I will change this in the next time.

**Review by webknjaz — APPROVED** [review]
